apps/core/components/searchForm/PlacePicker/PlacePickerContent.js

```diff
@@ -203,6 +203,19 @@ class PlacePickerContent extends React.Component<Props, State> {
     this.props.onClose();
   };
 
+  handleDeletePress = index => {
+    const locations = this.props[this.props.pickerType];
+
+    if (locations && locations.length !== 0) {
+      this.setState(state => ({
+        tempLocations: [
+          ...state.tempLocations.slice(0, index),
```

Comment:
what about use ramda for that?
https://ramdajs.com/docs/#remove

Reply:
Done